.circleci/config.yml

```diff
@@ -38,7 +38,7 @@ workflows:
               cluster_type: 
                 - s2ms
       - pytest:
-          name: "SingleStore 8.1 connection test"
+          name: "SingleStore 8.5 connection test"
```

Comment:
let's have both 8.1 and 8.5 tests (you'll need to change setup-cluster.sh to accept image as a parameter)